patches/relay-runtime+6.0.0.patch

```diff
@@ -0,0 +1,12 @@
+patch-package
+--- a/node_modules/relay-runtime/lib/store/RelayResponseNormalizer.js
++++ b/node_modules/relay-runtime/lib/store/RelayResponseNormalizer.js
+@@ -349,7 +349,7 @@ function () {
+         //
+         // Otherwise, missing fields usually indicate a server or user error (
+         // the latter for manually constructed payloads).
+-        if (process.env.NODE_ENV !== "production") {
++        if (process.env.NODE_ENV !== "production" && !(responseKey === "id" && storageKey === "id")) {
+           process.env.NODE_ENV !== "production" ? warning(this._isClientExtension || parent.kind === LINKED_FIELD && parent.concreteType == null ? true : Object.prototype.hasOwnProperty.call(data, responseKey), 'RelayResponseNormalizer: Payload did not contain a value ' + 'for field `%s: %s`. Check that you are parsing with the same ' + 'query that was used to fetch the payload.', responseKey, storageKey) : void 0;
+         }
+ 
```

Comment:
@alloy - Once the main bulk of the work is completed it would be nice if we could clean up our reliance on `patch-package` for relay / graphql related stuff.

Reply:
Aye, agreed–will add it to the post-migration list.

Reply:
Thank you!